src/App.tsx

```diff
@@ -1,56 +1,99 @@
-import React from 'react';
+import React, { useState, useMemo, useCallback } from 'react';
 import './App.scss';
+import cn from 'classnames';
 import { peopleFromServer } from './data/people';
+// import { Person } from './types/Person';
+
+function debounce(callback: () => void, delay: number) {
+  const timerID = 0;
+
+  return (...args: any) => {
+    window.clearTimeout(timerID);
+    window.setTimeout(() => {
+      callback(...args as []);
+    }, delay);
+  };
+}
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [value, setValue] = useState('');
+  const [nameIs, setNameIs] = useState<string | undefined>('Carolus Haverbeke');
```

Comment:
nameIs is not corrent, _is_ used as prefix for boolean variables
so, I suggest you to rename it just for name
check other places 
```suggestion
  const [name, setName] = useState<string | undefined>('Carolus Haverbeke');
```